docs/core/tutorials/cli-templates-create-template-package.md

```diff
@@ -28,96 +27,85 @@ In this part of the series you'll learn how to:
 
   This tutorial uses the two templates created in the first two parts of this tutorial. You can use a different template as long as you copy the template, as a folder, into the _working\templates\\_ folder.
 
-* Open a terminal and navigate to the _working\\_ folder.
+* Open a terminal.
```

Comment:
What folder should the terminal be navigated to before the next command is executed?

Reply:
This was changed because for next command (`dotnet new install`) the folder doesn't matter. 
The only thing that matters having nuget.org available in nuget.config so the package can be downloaded.

In step 2 (https://github.com/dotnet/docs/pull/35985#discussion_r1259984314) there is an instruction which folder to use.

Generally, we can keep the old wording and remove instruction later on - it doesn't matter much.

Reply:
> Generally, we can keep the old wording and remove instruction later on - it doesn't matter much.

If the folder you're in does have a bearing on the NuGet config file you're using, it makes sense to me to keep the old wording and remove the instruction later on.